compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -1858,6 +1858,15 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                         Applicability::MaybeIncorrect,
                     )),
                 )
+            } else if ident.name == sym::core {
+                (
+                    format!("maybe a missing crate `{ident}`?"),
+                    Some((
+                        vec![(ident.span, "std".to_string())],
+                        format!("try using `std` instead of `core`"),
```

Comment:
unnecessary format macro?

```suggestion
                        "try using `std` instead of `core`".to_string(),
```